packages/ia-pic-uploader/src/ia-pic-uploader.ts

```diff
@@ -202,7 +192,7 @@ export class IAPicUploader extends LitElement {
     document.addEventListener('dragover', e => this.dragOver(e), false);
     document.addEventListener('dragleave', e => this.dragLeave(e), true);
     document.addEventListener('drop', e => this.drop(e), false);
-    document?.addEventListener('saveProfileAvatar', (e: Event) => {
+    document.addEventListener('saveProfileAvatar', (e: Event) => {
```

Comment:
What is emitting this event?

Reply:
Is the `/account/settings` page emitting that event, then this component is listening for that event to start the saving process?

If that is the case, could we just call a method directly on this component since the settings page consumes this component? The event approach seems a bit roundabout way of communicating this intent.